examples/cd_chi2ks_adult.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<p>I would cleanup the outputs here as they are not relevant to the example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>